### PR TITLE
Add videos layout

### PR DESCRIPTION
**Pull request by lunson**
This is the proposed videos layout I mention in the May-13 email to @pwalczysko:
- It's more visually engaging that just using links
- It's mobile friendly
- Users can watch the video here without leaving the page

There was talk in one of the weekly OME meetings to perhaps use the same embedding method Glencoe Software used for their webinars: https://www.glencoesoftware.com/media/webinars/ so I included a sample Glencoe video here.

Someone had also mentioned perhaps using YouTube, so I included a sample (worth watching, I might add 😄) of that also.

Not ready for merge until we have the actual speaker videos, but I want to leave this here so you have something to work with in case those videos arrive after I've already left. (The `program_agenda.html` `include` file could probably use some improvement by someone who is more adept at `if` /`then` and `forloop` statements, as that is not my strength!)

Happy to take suggestions for improvement!

If this layout doesn't work for you guys, what was originally proposed should be just fine.

### CODE REVIEW

**Comment by lunson** [comment]
Not sure why the build isn't working, and I don't understand the error message...

**Comment by pwalczysko** [comment]
imho the travis failure is something rather deep, not concerning this PR. 

``The command "eval git clone --depth=50 https://github.com/ome/www.openmicroscopy.org.git ome/www.openmicroscopy.org " failed. Retrying, 2 of 3.``

It seems travis cannot even clone the repo, so I guess we need @sbesson or @joshmoore to have a look please.

**Comment by pwalczysko** [comment]
I restarted the build from https://travis-ci.org/github/ome/www.openmicroscopy.org/pull_requests - maybe it will simply start to behave

**Comment by pwalczysko** [comment]
It is sorted now, some mystery intermittent failure only

**Comment by pwalczysko** [comment]
Please see https://docs.google.com/document/d/1Sdvyqk87zTxKZ1fxMU2GkJZBZmwxLNRZvVsXNs0bEA4/edit?disco=AAAAJiwZq0A

File: events/ome-community-meeting-2020/day1/index.html [hunk 1]
```diff
@@ -10,15 +10,19 @@
       vid_title1: IDR
       vid_speaker1: "Sebastien Besson & Frances Wong"
       vid_aff1: OME Dundee
+      vid_link1: "https://static-movie-usa.glencoesoftware.com/webinars/glencoe-webinar-multi-image-workflows-with-pathviewer-grid.mp4?_ga=2.2888752.1473852673.1589360057-1663755810.1554828337"
```
Comment (pwalczysko) [inline events/ome-community-meeting-2020/day1/index.html:13]:
```suggestion
      vid_link1: "https://downloads.openmicroscopy.org/presentations/2020/Dundee/main-talks/main-talk-test.mp4?_ga=2.2888752.1473852673.1589360057-1663755810.1554828337"
```

**Comment by pwalczysko** [comment]
@lunson allowed myself to push a commit with the intention to test the way we would like to host the videos for the meeting.

**Comment by pwalczysko** [comment]
It seems to work, see https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2020/day1/

**Comment by pwalczysko** [comment]
@jrswedlow asks about some placeholder icons for the times when we deploy this PR but still do not have (all) the videos

**Comment by lunson** [comment]
A placeholder image has now been added for later video submissions.

**Comment by pwalczysko** [comment]
@lunson the placeholders are nice, we can go with them please (I understand that the placeholders are there for the videos 3 and 4 on day 1).
Please remove also the first video on day 1, that was just a test, thank you.

We hope to have some real videos soon :)

cc @jrswedlow

**Comment by lunson** [comment]
@pwalczysko Closing this PR for now, since Jason wants to see some other updates go live: https://trello.com/c/RIcmFL52/21-virtual-agenda#comment-5ec1694f8500f816922426e6

I'll reopen this branch again after rebasing and getting this as up-to-date as possible before my pending departure.

**Pull request closed without merging**